src/app/shared/models/action.ts

```diff
@@ -5,9 +5,21 @@ export interface Action {
   type: ActionType
   name: string
   description?: string
+  argument?: Argument
   metadata?: unknown
 }
 
+export interface Argument {
+  description: string
+  name: string
+  type: ActionArgumentSchema
+}
+
+export enum ActionArgumentSchema {
+  STRING = 'STRING',
+  NUMBER = 'NUMBER',
+}
```

Comment:
I think that Argument should be change to `ActionArgumentSchema` and the current `ActionArgumentSchema` should be changed to `ActionArgumentSchemaType`

Reply:
Done